tests/cypress/support/commands_opencv.js

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+let selectedValueGlobal = '';
+
+Cypress.Commands.add('opencvCreateShape', (opencvShapeParams) => {
+    if (!opencvShapeParams.reDraw) {
+        cy.get('body').focus();
+        cy.get('.cvat-tools-control').trigger('mouseleave').trigger('mouseout').trigger('mouseover');
+        cy.switchLabel(opencvShapeParams.labelName, 'opencv-control');
+        cy.get('.cvat-opencv-control-popover-visible').within(() => {
+            cy.get('.ant-select-selection-item').then(($labelValue) => {
+                selectedValueGlobal = $labelValue.text();
+            });
+        });
+        cy.get('.cvat-opencv-drawing-tool').click();
+    }
+    opencvShapeParams.pointsMap.forEach((element) => {
+        cy.get('.cvat-canvas-container').click(element.x, element.y);
+    });
+    if (opencvShapeParams.useDoneButton) {
+        cy.contains('span', 'Done').click();
+    } else {
+        const keyCodeN = 78;
+        cy.get('.cvat-canvas-container')
+            .trigger('keydown', { keyCode: keyCodeN })
+            .trigger('keyup', { keyCode: keyCodeN });
+    }
+    cy.opncvCheckObjectParameters('POLYGON');
+});
+
+Cypress.Commands.add('opncvCheckObjectParameters', (objectType) => {
+    cy.get('.cvat-opencv-control-popover').should('be.hidden');
```

Comment:
Let's remove this check. It might be changed in the future.

Reply:
Removed.